.gitignore

```diff
@@ -509,3 +509,6 @@ $RECYCLE.BIN/
 *.lnk
 
 # End of https://www.toptal.com/developers/gitignore/api/vim,git,macos,linux,pydev,emacs,dotenv,python,windows,webstorm,pycharm+all,jupyternotebooks
+
+# Integration tests cloud configs
+tests/integration/cloud-config-*.ini
```

Comment:
According to the comment on the top, this file is (or has been) generated automatically, but TBH I have not seen that happening. 

@felixfontein do we have any refresh process for this file?

That being said, it is usual to end the file with a newline character.

Reply:
This kind of files is required by `ansible-test` for targets flagged with `cloud` (specified in `aliases` file).
Since in the case of Scaleway modules it can store the API key, we should ignore the file to prevent pushing it to GitHub by mistake

Reply:
Oh, I was not aware of that "feature" in `ansible-test`. I find it debatable to have transient files (especially ones designed to contain sensitive information) to be placed _within_ the repository tree. I think that kind of data should be somewhere like, say, `${HOME}/.config/ansible-test/cloud-config-*.ini` or similar. But that's just me ranting.

I think the auto-generated aspect of the file is still a concern, although a somewhat minor one. If this files gets to be auto-generated again, this line will be lost.

Reply:
> I find it debatable to have transient files (especially ones designed to contain sensitive information) to be placed _within_ the repository tree. I think that kind of data should be somewhere like, say, `${HOME}/.config/ansible-test/cloud-config-*.ini` or similar. But that's just me ranting.

Since these files are usually collection specific, it does make sense to store them with the collection.

Right now there is no process of automatically updating `.gitignore`. If someone wants to update it, they should only update the parts inside the marked areas. Since this is coming after the maked areas,

Reply:
@felixfontein looks like we missed the rest of the sentence... ?

Reply:
The royal we is forgetful, it seems :D I've edited the comment.